tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MatchTypeSuite.scala

```diff
@@ -288,11 +288,11 @@ class MatchTypeSuite extends BaseDottySuite {
                     Type.Name("L")
                   ),
                   TypeCase(
-                    Type.AnonymousParam(None),
+                    Type.PatWildcard(),
                     Type.Name("R")
                   ),
                   TypeCase(
-                    Type.Wildcard(Type.Bounds(None, None)),
+                    Type.Name("?"),
```

Comment:
```suggestion
                    Type.PatWildcard("?"),
```

Both should be wildcards.

Reply:
same answer as before. `?` is a generic type parameter wildcard, but not a pattern.

Reply:
Currently, `case _ => R` means exactly the same as `case ? => R`, so why would it treat differently?

Reply:
means where? i created a Scala 3 test project and tried using `case ? => ...`, and the compiler complained that type `?` was not defined. it didn't look at it as a wildcard. i must misunderstand what you said, then.

Reply:
Actually, you are right. Didn't realize that `?` is limited to parameter types.